Source/UI/BillingForm/View/BillingFormTextFieldView.swift

```diff
@@ -80,6 +80,10 @@ class BillingFormTextFieldView: UIView {
 
         updateTextField(style: style, textFieldValue: textFieldValue, tag: tag)
         updateErrorView(style: style)
+        layoutIfNeeded()
+        layoutSubviews()
+        textFieldContainer.layoutIfNeeded()
+        textFieldContainer.layoutSubviews()
```

Comment:
I wonder if we really need this code on update. 

The objects lifecycle may invoke on update, so would be curious if we can instead just use object's own lifecycle updates:

```swift
override func layoutSubviews() {
    super.layoutSubviews()

    textFieldContainer.layoutIfNeeded()
    textFieldContainer.layoutSubviews()
}
```